Feature/matías

Mati279 opened the pull request.

Mati279 committed:
Initial Commit

Mati279 committed:
Initial Commit

Mati279 committed:
Initial Commit - Tarea Grupal 1

gpgpublickey commented on Trabajo Grupal 1/Mascota.cs:
```diff
@@ -0,0 +1,67 @@
+    {
+        public string Nombre { get; set; }
+        public int Edad { get; set; }
+        public Especie Especie { get; set; }
```
Interesante decisión de diseño, aunque rompe algunos principios SOLID de los cuales vamos a hablar en clase, me gustaria que intentes aplicar mejoras una vez los hayamos hablado. Pero aplicaste todos los conceptos, y el de la clase para especie fue MUY interesante ya vamos a ver adelante como usar esta estrategia de una forma SOLID

Mati279 committed:
Corrección Tarea

Tratando de usar SOLID.

Mati279 committed:
Delete gitignore.txt

Mati279 committed:
Programas individuales - Tarea de operadores

Hice los programas separados porque así estaba en la consigna, si preferís que los junte en un solo programa decime y lo hago! Me queda pendiente hacer la calculadora. Saludos.

Mati279 committed:
Calculadora SOLID
